runtime/ms-rest/test/retryPolicyTests.js

```diff
@@ -131,3 +132,165 @@ describe('exponentialretrypolicyfilter-unittests', function () {
     });
   });
 });
+
+describe('systemErrorRetrypolicyfilter-unittests', function () {
+  it('RetrySucceedsOn_ENOTFOUND_ErrorCode', function (done) {
```

Comment:
@huangpf - these are the unit tests that ensure that the systemErrorRetryFilter works as expected for sleect system errors.